src/components/DrawerTabPanels/DetailsTabPanel/AttributeSelectionTable.tsx

```diff
@@ -1,35 +1,32 @@
-import { Box, Button, Dialog, DialogContent, DialogTitle, DialogActions } from '@mui/material';
+import { Button, Dialog, DialogContent, DialogTitle, DialogActions } from '@mui/material';
 import * as React from 'react';
 import { useDispatch } from 'react-redux';
 import DataGrid, { SelectColumn } from 'react-data-grid';
 import { groupBy as rowGrouper } from 'lodash';
-import { RootState, usePSESelector } from '../../Store/Store';
-import genericFingerprintAttributes, { setGenericFingerprintAttributes } from '../../Ducks/GenericFingerprintAttributesDuck';
+import { usePSESelector } from '../../Store/Store';
 import { DefaultFeatureLabel } from '../../../model';
+import { setFips } from 'crypto';
 
-export function AttributeSelectionTable(props: any) {
+export function AttributeSelectionTable(props: { attributes: any[]; setAttributes: (attributes: any[]) => void }) {
```

Comment:
this should result in a lint error, never use props: ... always destructure in the function
{ attributes, setAttribute, children }: PropsWithChildren<{ attributes: Type, ... }>
also dont use any but type accordingly